### PR TITLE
IslyObject omit and pick modifiers

### PR DESCRIPTION
**Pull request by AntonCroner**

### CODE REVIEW

**Comment by tvartom** [comment]
Don't forget to update `README.md`. (And remove it from `.vscode/settings.json`-> `files.exclude` coz that is just stupid.)

**Comment by tvartom** [comment]
Methods should be possible to use without providing generics. (`type.omit(["a"])` should be the same as `type.omit<"a">(["a"])`).
Which I think it is, but there is no test for it.

**Comment by tvartom** [comment]
Also add a test with omit used on an extended object. Both removing from the direct parent, and from further down, to make sure this also works.

**Comment by tvartom** [comment]
I fixed the extended, rewrote the omit, added pick and updated readme.

**Review by tvartom — APPROVED** [review]

File: object.spec.ts [hunk 1]
```diff
@@ -175,4 +175,129 @@ describe("isly.object", () => {
...
+		// Extra properties may still be present:
+		expect(errorResponseWithoutBodyType.is({ status: 200, body: "payload" })).toBe(false)
+		expect(errorResponseWithoutBodyType.is({ status: 500, body: "payload" })).toBe(true)
+		expect(errorResponseWithoutBodyType.is({ status: 1000, body: "payload" })).toBe(false)
+
+		// get should not return omited property and return undefined if all conditions isn't met:
+		expect(errorResponseWithoutBodyType.get({ status: 200, body: "payload" })).toBeUndefined()
+		expect(errorResponseWithoutBodyType.get({ status: 500, body: "payload" })).toEqual({ status: 500 })
+		expect(errorResponseWithoutBodyType.get({ status: 1000, body: "payload" })).toBeUndefined()
+	})
```
Comment (marcushanses) [inline object.spec.ts:302]: Remove empty lines.

**Review by marcushanses — CHANGES_REQUESTED** [review]

**Review by marcushanses — APPROVED** [review]

**Pull request merged**